### ISSUE TITLE
Build mixin to pass DDPRateLimiter options

### ISSUE THREAD

**Issue by dan335** [meteor/validated-method#19]
Would it make sense to add DDPRateLimiter to validated-method?

**Comment by stubailo** [comment]
Yes, I think this should be part of #22 

But it would work right now, you just have to reference the method by name.

**Comment by stubailo** [comment]
Added mixins in Validated Method 1.0:

https://github.com/meteor/validated-method#mixins

Now it's just a matter of writing a rate limiter one!

**Comment by stubailo** [comment]
If someone wants to try, it will look very similar to this one: https://github.com/stubailo/meteor-rest/blob/devel/packages/rest-method-mixin/rest-method-mixin.js

**Comment by sys13** [comment]
I made some code for the mixin but I realized that I can't even get regular rate-liming working. [Posted on the forums](https://forums.meteor.com/t/has-anyone-gotten-ddp-rate-limiter-working/16562) with an [example reproduction](https://github.com/sys13/rate-limiting).

**Comment by stubailo** [comment]
OK just replied to that thread and fixed the docs. If you want to see a working example of DDPRateLimiter, see the example app for the guide: https://github.com/meteor/todos/blob/001b6eafdd83e2a94b425561e8d1d4b5146bfbb0/imports/api/todos/methods.js#L106

**Comment by sys13** [comment]
Thanks for fixing the docs. I think I'm going to go with [Sikka](https://github.com/meteorhacks/sikka) instead as it seems to do a better job at protecting against DDOS attacks.

**Comment by stubailo** [comment]
Yes, the goal of ddp rate limiter is to protect against someone brute forcing your data but doesn't do anything about DDOS.

**Comment by nlhuykhang** [comment]
I created a mixin as npm package [ddp-rate-limiter-mixin](https://github.com/nlhuykhang/ddp-rate-limiter-mixin) here. Could I create an PR to include it in README?
